flowcraft/generator/inspect.py

```diff
@@ -1465,8 +1465,8 @@ def _prepare_static_info(self):
         return pipeline_files
 
     def _dag_file_to_dict(self):
-        """Function that opens the dotfile named .treeDag.json in the current
-        working directory
+        """Function that opens the accessory named treeDag.json in the
```

Comment:
What is accessory? Also notice that this method not only opens that file but loads its content to a dict and hence the docstring is incomplete. It was already incomplete before I know.